common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -67,6 +67,34 @@ public static X509TrustManager getDefaultX509TrustManager() throws KeyManagement
     }
 
     /**
+     * Utility that exposes common TLS constants.
+     */
+    public static final class Constants {
+        private Constants() {}
+
+        /**
+         * Returns the maximum length (in bytes) of plaintext data.
+         */
+        public static int maxPlaintextLength() {
+            return NativeConstants.SSL3_RT_MAX_PLAIN_LENGTH;
+        }
+
+        /**
+         * Returns the maximum length (in bytes) of an encrypted packet.
+         */
+        public static int maxEncryptedPacketLength() {
+            return NativeConstants.SSL3_RT_MAX_PACKET_SIZE;
+        }
+
+        /**
+         * Returns the length (in bytes) of an encrypted packet header.
```

Comment:
I actually think we may not need to expose the header length. Let me remove it for now.